content/docs/setup/kubernetes/multicluster-install/flat-network/index.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: Istio Multicluster
```

Comment:
To a more understandable/user-friendly terminology.

Reply:
Not clear that "Using VPN/RFC1918 network" is any easier to understand than the more straightforward multicluster install.  In addition, multicluster is used all over our documentation and references the single control plane model, not just this one document.  Those documents need a reorg in the same way as the "multicloud" subtree does.

Reply:
@rshriram Please take a look at #2703 
I am already addressing these changes there. Feel free to leave suggestions.
IMO, it would be best if you left the changes to this file out of this PR. We can address them in #2703

Reply:
The environments working group has already debated and accepted the nomenclature. (Multicloud = umbrella for all cloud-connected related tech.  Within that umbrella there are currently 3 technologies:
- multicluster
- mesh expansion
- zerovpn

The environments workgroup knew a year ago there would be multiple technologies in this space so we gave everything an "umbrella" term and let everyone pick names for their particular focus area.  This is how we have 3 terms - in addition to split horizon and possibly other terms coming.  If you like, I can submit a PR that properly breaks out the umbrella terminology from the technologies, however, you seem to be doing that here, so it would be easier on me if you did so.

We can of course have this nomenclature discussion again - in the environments workgroup meeting is where these things are decided.

Cheers
-steve

Reply:
The environments working group is not omniscient :). Second, the choice of these terms is very un-user friendly.. No one cares about split or joint horizon or zero/with vpn. What people care about is connecting two clusters. And our docs should address that need rather than differentiating the various implementation nuances with new terminologies. Also keep in mind that the people in the working group a year ago are not there today. The public perception of this content is important. With constant complaint of bloat, complexity and unnecessary feature add, its important to water down the concepts and stick to the base realities rather than coin new terms for "tech" that people can use. [we did not have this problem a year ago].

The docs should not dictate how the concepts are positioned in various presentations/conferences, nor should the implementation dictate the exposition of fundamental building blocks in docs.

You could add blogs, tutorials and other advanced stuff that can use all the terms coined in that one-year old meeting. See how Vadim is going about the same with Egress. The tasks are focussed on the purpose -- limiting/observing external traffic, not on jargons like "firewalling external traffic", "establishing a software DMZ", or "connecting to other clouds", etc. It simply describes the what and how.